logger.go

```diff
@@ -0,0 +1,27 @@
+package bigcache
+
+import (
+	"log"
+	"os"
+)
+
+// Logger is invoked when `Config.Verbose=true`
+type Logger interface {
+	Printf(format string, v ...interface{})
+}
+
+var _ Logger = &log.Logger{}
```

Comment:
Why do we need it?

Reply:
This will emit a compile error in case stdlib's `log.Logger` fails to implement our logger interface.

https://golang.org/doc/faq#guarantee_satisfies_interface

Reply:
OK, that's what I suspected. Can you add a comment on this with a link. Just in case somebody wants to delete this line in a future.

@medzin What do you think?